pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -587,6 +590,19 @@ type AlertmanagerLimitsSpec struct {
 	MaxPerSilenceBytes *ByteSize `json:"maxPerSilenceBytes,omitempty"`
 }
 
+// AlertmanagerDispatchSpec defines the dispatch command line flags when starting Alertmanager.
+// +k8s:openapi-gen=true
+type AlertmanagerDispatchSpec struct {
```

Comment:
Hi @nutmos I don't think you need to create a separated type for this, could just be directly under the am spec.